src/CommandListManager.cpp

```diff
@@ -26,7 +26,7 @@ namespace D3D12TranslationLayer
         , m_bNeedSubmitFence(false)
         , m_pCommandList(nullptr)
         , m_pCommandAllocator(nullptr)
-        , m_AllocatorPool(false /*bLock*/)
+        , m_AllocatorPool(false /*bLock*/, cMaxInFlightDepth[(size_t)type]/* max allocators in use until wait/block for completion*/)

```

Comment:
I don't think that comment is necessary, the array (or as suggested, function) name is fine.